packages/apps/plugins/plugin-status-bar/src/components/StatusBarImpl.tsx

```diff
@@ -0,0 +1,45 @@
+//
+// Copyright 2024 DXOS.org
+//
+
+import { DiscordLogo, Lightning, PaperPlaneTilt } from '@phosphor-icons/react';
+import React from 'react';
+
+import { Popover } from '@dxos/react-ui';
+
+import { FeedbackForm } from './FeedbackForm';
+import { StatusBar } from './StatusBar';
+
+export const StatusBarImpl = () => {
+  const [open, setOpen] = React.useState(false);
+
+  return (
+    <Popover.Root open={open} onOpenChange={setOpen}>
+      <StatusBar.Container>
+        <StatusBar.EndContent>
+          <Popover.Trigger asChild>
+            <StatusBar.Button>
+              <PaperPlaneTilt />
+              <StatusBar.Text classNames='hidden sm:block'>Feedback</StatusBar.Text>
+            </StatusBar.Button>
+          </Popover.Trigger>
+          {/* TODO(Zan): Configure this? */}
+          <a href='https://discord.gg/zsxWrKjteV' target='_blank' rel='noopener noreferrer'>
+            <StatusBar.Button>
+              <DiscordLogo />
+              <StatusBar.Text classNames='hidden sm:block'>Discord</StatusBar.Text>
+            </StatusBar.Button>
+          </a>
```

Comment:
`Statusbar.Button` should support `asChild` so these don’t need to nest like this.